ASU/asu-unity-stack
# Percy visual regression testing

### Description
Added percy visual regression testing with story config

TESTING STEPS:
If running tests locally and only wanting to check a dry run, you must set `PERCY_PARALLEL_NONCE=<random alphanumeric string>` and add the --dry-run flag to the percy-testing.js script. If you do not have the token set in your local environment, you can safely run this without the dry run flag as it will not upload but all other steps will run.


### Links

- [Unity reference site](https://unity.web.asu.edu/)
- [JIRA ticket](https://asudev.jira.com/browse/UDS-1210?atlOrigin=eyJpIjoiMTQ1NTZmMDYyY2RiNDY2ZmFhYTI3OTc5NDI0MjAxYjMiLCJwIjoiaiJ9)

### Checklist

- [x] Unity project successfully builds from root `yarn install` & `yarn build`
- [x] Commits do not contain multiple scopes
- [x] Add/updated examples
- [x] Add/updated READMEs/docs
- [x] Accessibility checks

### Browsers


### Images
